src/benchmarklib/benchmark_runner.cpp

```diff
@@ -198,9 +199,14 @@ void BenchmarkRunner::_benchmark_ordered() {
 
     const auto& name = _benchmark_item_runner->item_name(item_id);
     std::cout << "- Benchmarking " << name << std::endl;
-
+    // clear results
+    _results[item_id].successful_runs = {};
+    _results[item_id].unsuccessful_runs = {};
+    _results[item_id].duration = {};
     auto& result = _results[item_id];
 
+    SegmentAccessStatistics_T::reset_all(Hyrise::get().storage_manager.tables());
```

Comment:
Passing the table vector into reset_all seems a bit weird.

Reply:
I'll ask you in person.